bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from cowsay import char_names, get_output_string
+from discord import HTTPException
+from discord.ext import commands
+
+from bot.bot import Bot
+
+# Creates a copy of supported cowsay character names to filter out characters which break the embed.
+SUPPORTED_CHAR_NAMES = list(char_names)
+for element in ["dragon", "trex", "stegosaurus", "turkey", "ghostbusters", "turtle"]:
+    SUPPORTED_CHAR_NAMES.remove(element)
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(SUPPORTED_CHAR_NAMES)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: Optional[str]) -> None:
+        """Builds a cowsay string and sends it to Discord."""
+        character = character.lower()
+        if not text:
+            text = f"I'm a {character}"
+
+        if "```" in text:
+            text = text[:1] + "\u200b" + text[1:]
+            text = text[:-2] + "\u200b" + text[-2:]
```

Comment:
This wouldn't stop the codeblock from breaking
```suggestion
        text = text.replace("`", "`\u200b")
```

Reply:
@ToxicKidz It does actually

Reply:
it looks bad though, ill do this